packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -26,6 +26,7 @@
     --movable-line-stroke-color: var(--mafs-blue);
     --movable-line-stroke-weight: 2px;
     --movable-line-stroke-weight-active: 4px;
+    overflow: initial;
```

Comment:
Yet another possible solution: `position: absolute` on the `svg` child of `.MafsView` fixes it for me (in Edge). I think that only works, though, because the parent of `.MafsView` happens to have `position: relative`. So to make this not context-dependent, I think we'd want `.MafsView` to have `position: relative`.